File: server/fetchMovies.js
const url = `https://api.themoviedb.org/3/movie/popular?api_key=0689392230f096b932bb1093687cce94&language=en-US&page=1`;
const axios = require('axios');

const fetchMovies = async () => {
	try {
		const response = await axios.get(url);
		const result = response.data.results;
		console.log(result[0]);
		return result;
	} catch (err) {
		console.log(err);
	}
};

fetchMovies();

File: pythonScript/fetchMovies.py
import requests

api_key = ''
url = 'https://api.themoviedb.org/3/movie/popular'


response = requests.get(url, params={'api_key': api_key, 'language': 'en-US', 'page': 1})

with open("data.json", "w") as file:
    file.write(requests.get(url, params={'api_key': api_key, 'language': 'en-US', 'page': 1}).text)

if response.status_code == 200:
    data = response.json()
    print(data)
else:
    print(f'Error: {response.status_code}')

